packages/vector_graphics_compiler/lib/src/geometry/matrix.dart

```diff
@@ -223,3 +227,239 @@ class AffineMatrix {
 [ 0.0, 0.0, 1.0 ] // _m4_10 = $_m4_10
 ''';
 }
+
```

Comment:
This makes me sad - I tried to get it right, I promise.

Reply:
ITS OK DAN. 😃